src/grid_map_geo.cpp

```diff
@@ -58,20 +58,20 @@
 #include <gdal/ogr_spatialref.h>
 #endif
 
-GridMapGeo::GridMapGeo() {}
+GridMapGeo::GridMapGeo(const std::string frame_id) { frame_id_ = frame_id; }
```

Comment:
```suggestion
GridMapGeo::GridMapGeo(const std::string& frame_id) { frame_id_ = frame_id; }
```


Pass by const-ref because it's a non-primitive type